forge/csrc/autograd/autograd.cpp

```diff
@@ -371,7 +371,7 @@ void autograd_engine::create_backward_graph(const grad_map &requires_grad_map)
 
                     NodeContext src = last_out;
                     last_out = create_op(
-                        OpType("reduce_sum", {dim}),
+                        OpType("reduce_sum", {dim}, {}, {{"keep_dim", true}}),
```

Comment:
Can we extract `keep_dim` from attributes? 

Getting this data from attributes will make our life easier when we go for support of keep_dim = False :))

Same comment for other occurrences :))